.github/workflows/install-index-release.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
```

Comment:
Please give this workflow a more descriptive name. Is this going to be used as a sort of end-to-end test?